### ISSUE TITLE
Installer silently skips installing prerequisites if internet access is blocked on the target machine

### ISSUE THREAD

**Issue by andreasohlund** [Particular/ServiceControl#3277]
We have a user report of the prerequisites silently not being installed if internet access is blocked

https://discuss.particular.net/t/servicecontrol-failedimports-and-ravendb-persister-3-5-vs-5/3293/4

**Comment by ngallegos** [comment]
This bug is no longer relevant because the SCMU installer no longer exists - it is now a standalone executable.
